compiler/codegen/OMRMachine.hpp

```diff
@@ -119,6 +122,29 @@ class OMR_EXTENSIBLE Machine
             return -1;
          }
       }
+
+   /**
+    * \brief Retrieve a pointer to the register file
+    */
+   TR::RealRegister **registerFile() { return _registerFile; }
+
+   /**
+    * \brief Retrieves the TR::RealRegister object for the given real register
+    *
+    * \param[in] rn : the desired real register
+    *
+    * \return : the desired TR::RealRegister object
+    */
+   TR::RealRegister *realRegister(TR::RealRegister::RegNum rn) { return _registerFile[rn]; }
```

Comment:
This seems to deviate from the rest of the codebase which the majority of uses the `get*` syntax for getters. Personally if I am loking t oretrieve something from a class I type `get` in my IDE and find the right getter which I want. This would definitely stump me. I am ok with the change as long as this is the direction we're heading in. Thoughts?

Reply:
It seems we are indeed to quote @0xdaryl consistently inconsistent in the area of naming these getters. I'm fine with the change as is. However I look forward to the other two reviews being addressed.